src/test/java/com/cloudant/tests/AttachmentsTest.java

```diff
@@ -85,12 +90,17 @@ public void attachmentInline_getWithDocument() {
     }
 
     @Test
-    public void attachmentStandalone() throws IOException {
+    public void attachmentStandalone() throws IOException, URISyntaxException {
         byte[] bytesToDB = "binary data".getBytes();
         ByteArrayInputStream bytesIn = new ByteArrayInputStream(bytesToDB);
         Response response = db.saveAttachment(bytesIn, "foo.txt", "text/plain");
 
-        InputStream in = db.find(response.getId() + "/foo.txt");
+        Document doc = db.find(Document.class, response.getId());
+        assertTrue(doc.getAttachments().containsKey("foo.txt"));
+
+        HttpConnection conn = Http.GET(new URI(db.getDBUri() + "/" + response.getId() + "/foo.txt"));
```

Comment:
Is this use of string concatenation intentional? I wonder why you didn't use `DatabaseURIHelper` here, or did you do this so that there is an alternative code path for building up the URL and this serves to check that the URL was encoded correctly by our production code?